docs/lib/graphqlapi/fragments/native_common/authz/common.md

```diff
@@ -1,9 +1,29 @@
 For client authorization AppSync supports API Keys, Amazon IAM credentials, Amazon Cognito User Pools, and 3rd party OIDC providers. This is inferred from the `amplifyconfiguration.json` file when you call `Amplify.configure()`.
 
-## API Key
+#### API Key
 
-API Key is the easiest way to setup and prototype your application with AppSync.
+API Key is the easiest way to setup and prototype your application with AppSync. 
+
+#### Cognito User Pools
```

Comment:
User Pools and Identity Pools are actually lower cased in their docs. By convention, refer to the service as Amazon Cognito in every mention.

Reply:
updated all places to use `user pools` and `identity pools`